docs/index.md

```diff
@@ -154,10 +154,19 @@ Snowfakery builds on a tool called SQLAlchemy, so it gets a variety of database
 
 When integrated with CumulusCI (see [Advanced Features](#advanced-features)) it is possible to output to a Salesforce instance.
 
-Snowfakery can also output JSON, directories of CSV and object diagrams.
+Snowfakery can also output JSON, SQL, directories of CSV and object diagrams.
 
 CSV output goes to a directory with one CSV file per table and a JSON manifest file in the [csvw](https://www.w3.org/TR/tabular-data-primer/) format.
 
+The complete list of file-based (as opposed to database-based) formats are:
+
+- JSON - a custom JSON dialect
+- TXT - debugging-style output)
+- CSV - a directory of CSV files plus a csvw file
+- SQL - a bulk import SQL file
```

Comment:
"bulk import"?

Reply:
Would "bulk insert" work for you? During the editing process for my blog post the writer pointed out that many people think of SQL as a "query language" and therefore it helps to clarify to be more specific.

Reply:
fc94b40

Reply:
Yes using "insert" is clearer, thanks